node/core/pvf/src/prepare/queue.rs

```diff
@@ -225,19 +220,18 @@ async fn handle_to_queue(queue: &mut Queue, to_queue: ToQueue) -> Result<(), Fat
 async fn handle_enqueue(
 	queue: &mut Queue,
 	priority: Priority,
-	pvf_with_params: PvfWithExecutorParams,
-	preparation_timeout: Duration,
+	pvf: PvfExhaustive,
 ) -> Result<(), Fatal> {
 	gum::debug!(
 		target: LOG_TARGET,
-		validation_code_hash = ?pvf_with_params.code_hash(),
+		validation_code_hash = ?pvf.code_hash(),
 		?priority,
-		?preparation_timeout,
+		preparation_timeout = ?pvf.timeout(),
```

Comment:
nit: Possible to log the whole `pvf` struct?

Reply:
With the code blob? :scream: 
IIUC, we need the validation code hash to be named exactly `validation_code_hash` in logs to be observable. The executor parameters are logged on `debug` level in the candidate validation subsys when validation is requested. So the only meaningful field here is `timeout`, I believe.

Reply:
Ah gotcha. I just thought it could be more future-proof in case more fields were added. Maybe it could impl `Display`, which would only print a very truncated code blob? Just an idea, feel free to resolve if it's a bad one.